From f363d3614885f3aa68f0830df0ddb7fcbbb62cb4 Mon Sep 17 00:00:00 2001
From: Judith Mendez <jm@ti.com>
Date: Thu, 6 Feb 2025 17:24:36 -0600
Subject: [PATCH 01/68] TI: arm: multi_v7_defconfig: Enable Display

Add missing configs for AM335x and AM437x platforms
to enable display, as modules.

PWM output from ecap used as backlight source, Hence enable
CONFIG_PWM_TIECAP
CONFIG_PWM_TIEHRPWM
along with the panel driver with
CONFIG_DRM_TILCDC

Signed-off-by: Praneeth Bajjuri <praneeth@ti.com>
Signed-off-by: Judith Mendez <jm@ti.com>
Signed-off-by: Krunal Bhargav <k-bhargav@ti.com>
Signed-off-by: Randolph Sapp <rs@ti.com>
---
 arch/arm/configs/multi_v7_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/multi_v7_defconfig b/arch/arm/configs/multi_v7_defconfig
index be89511c3e26..fb0b141ce53c 100644
--- a/arch/arm/configs/multi_v7_defconfig
+++ b/arch/arm/configs/multi_v7_defconfig
@@ -735,6 +735,7 @@ CONFIG_DRM_RCAR_DU=m
 CONFIG_DRM_SUN4I=m
 CONFIG_DRM_OMAP=m
 CONFIG_OMAP5_DSS_HDMI=y
+CONFIG_DRM_TILCDC=m
 CONFIG_DRM_MSM=m
 CONFIG_DRM_FSL_DCU=m
 CONFIG_DRM_TEGRA=y
@@ -1208,6 +1209,8 @@ CONFIG_PWM_ATMEL_TCB=m
 CONFIG_PWM_BCM2835=y
 CONFIG_PWM_BRCMSTB=m
 CONFIG_PWM_FSL_FTM=m
+CONFIG_PWM_TIECAP=m
+CONFIG_PWM_TIEHRPWM=m
 CONFIG_PWM_MESON=m
 CONFIG_PWM_RCAR=m
 CONFIG_PWM_RENESAS_TPU=y
-- 
2.47.2

